Geometry/HcalCommonData/interface/HcalDDDRecConstants.h

```diff
@@ -78,9 +79,9 @@ class HcalDDDRecConstants {
   HcalID                    getHCID(int subdet, int ieta, int iphi, int lay,
 				    int idepth) const;
   std::vector<HFCellParameters>    getHFCellParameters() const;
-  void                      getLayerDepth(int ieta, std::map<int,int>& layers) const;
-  int                       getLayerFront(int det, int eta, int phi, int depth) const;
-  double                    getLayer0Wt(int det, int phi, int zside) const {return hcons.getLayer0Wt(det,phi,zside);}
+  void                      getLayerDepth(const int ieta, std::map<int,int>& layers) const;
```

Comment:
Could you merge the PR. I shall update this later on. The merged code is needed by many these days and is needed for the Barrel TDR studies